clang/test/SemaSYCL/Inputs/private_alloca.hpp

```diff
@@ -2,6 +2,8 @@
 
 #include "./sycl.hpp"
 
+#include <stddef.h>
```

Comment:
I'm not sure about adding stddef to lit tests. @premanandrao @AaronBallman can you comment here?

Reply:
Other tests in `SemaCXX` are doing this. I don't think this is an issue

Reply:
`stddef.h` is generally fine to use in lit tests because it's provided by the compiler and it does not do an `include_next` to something on the system. (Basically, we want lit tests to be hermetic and not rely on anything about the test system, so in general it's not okay to include system headers outside of passing `-ffreestanding` and using the compiler ones, but this case is a bit special.)

Reply:
Cool. Thanks! In that case, we can use this header in particular.